Content.Shared/Store/CurrencyPrototype.cs

```diff
@@ -0,0 +1,28 @@
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+
+namespace Content.Shared.Store;
+
+/// <summary>
+///     Prototype used to define different types of currency for generic stores.
+///     Mainly used for antags, such as traitors, nukies, and revenants
+///     This is separate to the cargo ordering system.
+/// </summary>
+[Prototype("currency")]
+[DataDefinition, Serializable, NetSerializable]
+public sealed class CurrencyPrototype : IPrototype
```

Comment:
Fields need a doc. Not the name and ID though those are fine due to being **genuinely** self-explanatory.